src/Microsoft.Performance.Toolkit.Engine/PluginSet.cs

```diff
@@ -355,7 +355,7 @@ public static PluginSet Load(string extensionDirectory)
                 var factory = new DataExtensionFactory();
                 repo = factory.CreateDataExtensionRepository();
 
-                var reflector = new DataExtensionReflector(assemblyDiscovery, repo);
+                var reflector = new DataExtensionReflector(assemblyDiscovery, repo, logger ?? Logger.Null);
```

Comment:
On line 344, we call `new AssemblyLoader()`, which does `Runtime.Logger.Create<AssemblyLoader>`. We might wat to do the same thing there

Reply:
I'm actually going to do the reverse and use Logger.Null if not specified so that there's a consistent, don't use console by default, behavior.

Reply:
That's what I meant 😄